### DISCUSSION TITLE
Code Review Deadline

### DISCUSSION THREAD

**Discussion by jennyhickson** [MetOffice/simulation-systems#242] [Announcements]
Hello,

**Changes wishing to become part of UM vn13.7, LFRic Apps vn1.2 and JULES vn7.7 need to be in code review by **20th September**** (note this does not mean that the review should be finished by then). 

It is advised that changes are in SciTech review by the 13th September to give reviewers a chance to look at them before the 20th. 

We are going to trial a subtle change in the use of ticket milestones in LFRic Apps, so please set your ticket appropriately:
 * "vn1.2" for changes that are a high priority for the vn1.2 release 
 * "next_release" for changes that need to be lodged as soon as possible but are not specifically required for vn1.2. 

We will prioritise reviewing tickets at the vn1.2 milestone and then acheive reviews of as many next_release tickets as possible before the release (which will shortly follow the UM release at the end of October). We will continue to accept requests for review of "next_release" tickets after the 20th September. The aim is to get a clearer sense of where tickets are targeted and remove the review freeze for hopefully the majority of tickets simply working through the process of getting onto trunk. 

Any questions on the process just get in touch,

Thanks
Jenny, on behalf of the SSD team

**Comment by james-bruten-mo** [comment]
Hi All,

A reminder that the CR deadline is end of this week - please get your tickets into CR by then.
We'll continue assigning reviewers for LFRic Apps with the `next_release` milestone after the deadline, although these will be reviewed at a lower priority.

Please get in touch with any worries.

James
